layouts/shortcodes/keps-data.html

```diff
@@ -0,0 +1,81 @@
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.css">
+
+<script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.js"></script>
+<script type="text/javascript">
+  $(document).ready( function () {
+    $('#keps-table').DataTable({
+      lengthMenu: [
+        [10, 25, 50, -1],
+        [10, 25, 50, 'All'],
+      ],
+    });
+});
+</script>
```

Comment:
I'd move this to the `<head>` of the page layout that gets used to render this page as HTML, and put it behind a `.HasShortcode` check.